README.md

```diff
@@ -12,7 +12,7 @@ This Terraform module deploys an S3-hosted static site with HTTPS enabled.
 ## Usage
 ```hcl
 module "s3_site" {
-  source    = "github.com/byu-oit/terraform-aws-s3staticsite?ref=v3.0.1"
+  source    = "github.com/byu-oit/terraform-aws-s3staticsite?ref=v3.0.2"
```

Comment:
I would recommend this be a major release (v4.0.0) since we are removing functionality. I doubt it would break anything, but IMO better safe than sorry.

Reply:
Done